### ISSUE TITLE
Formatting equations (and almost everything else) in LaTeX also uses packages

### ISSUE THREAD

**Issue by ttimbers** [swcarpentry/modern-scientific-authoring#14]
Here:

_"And if that wasn't confusing enough: HTML and Markdown do not support equations directly, but packages exist to allow authors to embed LaTeX-style equations in documents of either kind. The Jupyter Notebook relies on one such package, which allows users to put equations and other things in Markdown cells to be rendered in the browser."_

It is pointed out that HTML and Markdown don't support equations directly, and require extra package management. Although equations may work in base LaTeX, packages are frequently used to format them. You also commonly use packages for pretty much everything else in LaTeX to get a really nice looking document... This isn't mentioned anywhere.

**Comment by gvwilson** [comment]
Good point - LaTeX package management is certainly a big argument 
_against_ it.

**Comment by wking** [comment]
On Wed, Feb 10, 2016 at 10:16:02AM -0800, Greg Wilson wrote:

> Good point - LaTeX package management is certainly a big argument 
> _against_ it.

… if you live on a system with a crummy package manager or no package
manager at all ;), in which case I guess you use TeX Live [1](https://www.tug.org/texlive/).  If you
have a good package manager, LaTeX package management is really easy.

**Comment by ttimbers** [comment]
Agreed @wking, but even with a good package manager it is another layer of complexities for newbies.

**Comment by wking** [comment]
On Wed, Feb 10, 2016 at 10:27:03AM -0800, Tiffany Timbers wrote:

> … even with a good package manager it is another layer of
> complexities for newbies.

They'll still need to install something, so swapping in a fatter
(virtual) package doesn't make things much more complicated.  On
Gentoo:

  # USE="extra science humanities" emerge -av app-text/texlive

will get you a pretty comprehensive kitchen sink, including (among
many, many other things) BibTeX.

**Comment by sprossiter** [comment]
I suspect one could write a whole essay on the state of LaTeX and how 'best practice' (at least for many scientific papers) requires knowledge and use of many packages; things like the (slow) [LaTeX3 project](https://latex-project.org/latex3.html) and let's-sort-it-out-ourselves-in-the-interim efforts like the [memoir package](https://www.ctan.org/pkg/memoir?lang=en) are relevant here, though I'm no expert. Plus surprising-to-newcomers things such as the relative difficulty of tables in LaTeX (over and above the shift to a compiled marked-up-source approach with a content-not-typesetting philosophy). Maybe some of that could be condensed into a sentence or two?

It has also seemed to me that there is a growing 'battle' in the text-based-source camp between LaTeX and Markdown/Pandoc as the standards for scientific documentation (or at least different sectors of it), as hinted at in issue #12. Is something on that worth putting in more explicitly? (Obviously that also makes it more difficult for the 'WYSIWYG switcher' since there is no obvious single path.)
